### PR TITLE
Fix image size limit

### PR DESCRIPTION
**Pull request by gskril**
Currently the frog debugger shows an error if an image is >256kb, but the [Farcaster docs](https://docs.farcaster.xyz/reference/frames/spec#images) say an image can be up to 10mb

image

### CODE REVIEW

**Review by tmm — COMMENTED** [review]
Nice catch. Can you add a changeset with `pnpm changeset`?

File: src/dev/components/Data.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function Data() {
             postUrl: 256,
             inputText: 32,
             state: 4_096,
-            image: 256,
+            image: 10_000,
```
Comment (jxom) [inline src/dev/components/Data.tsx]: think you got kb and mb mixed up

```suggestion
            image: 10_485_760,
```

File: src/dev/components/Data.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export function Data() {
             postUrl: 256,
             inputText: 32,
             state: 4_096,
-            image: 256,
```
Comment (tmm) [inline src/dev/components/Data.tsx:17]: Data URIs are 256 so probably makes sense to check the image and either set the limit conditionally.

**Comment by tmm** [comment]
Fixed when devtools got rewritten #156

**Pull request closed without merging**